gno.land/pkg/keyscli/addpkg.go

```diff
@@ -0,0 +1,140 @@
+package keyscli
```

Comment:
please, add a small readme explaining the goal of this package and its relationships and add a link to this PR.